app/controllers/decidim/user_exporter/admin/user_exporter_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  module UserExporter
+    module Admin
+      class UserExporterController < Decidim::Admin::ApplicationController
+        def export_users
+          enforce_permission_to :read, :officialization
+
+          ExportUsers.call(params[:format], current_user) do
+            on(:ok) do |export_data|
+              ExportUsersJob.perform_now current_user, "users", export_data
+              flash[:notice] = t("notice", scope: "decidim.admin.exports")
```

Comment:
Should not the job be in the command?

Reply:
Why not calling the export mailer directly instead of using a job which calls the mailer ?